subprojects/gradle/instrumentation-tests/src/main/kotlin/com/avito/instrumentation/report/Report.kt

```diff
@@ -17,6 +17,8 @@ import java.io.Serializable
 
 interface Report : ReadReport {
 
+    val reportApi: ReportsApi
```

Comment:
Where could I look to realize why we need report API there?

Reply:
let me try to remove this (was a refactoring leftover)